craft_parts/errors.py

```diff
@@ -12,7 +12,7 @@
 # Lesser General Public License for more details.
 #
 # You should have received a copy of the GNU Lesser General Public License
-# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+# along with this program.  If not, see <<http://www.gnu.org/licenses/>.
```

Comment:
There's a double `<` here.

Reply:
Thanks! I must have typoed that when I was looking for the branch difference in that file (the only conflict). Fixed!